verilog/formatting/formatter_test.cc

```diff
@@ -5633,7 +5631,7 @@ static constexpr FormatterTestCase kFormatterTestCases[] = {
     {"module mp ;property p1 ; a|->## [0  : 1]  b;endproperty endmodule",
      "module mp;\n"
      "  property p1;\n"
-     "    a |-> ##[0 : 1] b;\n"  // limit to one space, symmetrize
+     "    a |-> ##[0:1] b;\n"  // compact bit slice range
```

Comment:
Actually, this is a (cycle?) delay range, not a bit slice, nor declared dimensions.
If you look at its CST, is it distinguishable from indexing and declared dimensions?
If it is, can we update the logic to exclude this case?
If it is not, we should consider re-tagging these CST nodes to distinguish them semantically.
[Example of a re-tagging a CST subtree.](https://cs.opensource.google/verible/verible/+/master:verilog/parser/verilog.y;drc=259321c5bb95882d2ac872ca03de6c8d35fca733;l=1897)

Reply:
N/A anymore in this PR (removed).